adapters/InMobi/InMobiAdapter/GADMAdapterInMobiUnifiedNativeAd.m

```diff
@@ -23,408 +23,424 @@ @interface GADMAdapterInMobiUnifiedNativeAd () <IMNativeDelegate>
-  completed();
 }
 
 #pragma mark - Async Image
 
 - (void)loadImageWithURL:(nonnull NSURL *)url
               imageCache:(nonnull NSCache *)imageCache
                 callback:(void (^)(UIImage *))callback {
-  dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_HIGH, 0), ^{
-    NSString *cacheKey =
+    dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_HIGH, 0), ^{
+        NSString *cacheKey =
         [url.absoluteString stringByAddingPercentEscapesUsingEncoding:NSUTF8StringEncoding];
-    UIImage *cachedImage = [imageCache objectForKey:cacheKey];
-    if (!cachedImage) {
-      NSData *imageData = [NSData dataWithContentsOfURL:url];
-      UIImage *image = [UIImage imageWithData:imageData];
-      if (image) {
-        cachedImage = image;
-        GADMAdapterInMobiCacheSetObjectForKey(imageCache, cacheKey, cachedImage);
-      }
-    }
-    dispatch_async(dispatch_get_main_queue(), ^{
-      callback(cachedImage);
+        UIImage *cachedImage = [imageCache objectForKey:cacheKey];
+        if (!cachedImage) {
+            NSData *imageData = [NSData dataWithContentsOfURL:url];
+            UIImage *image = [UIImage imageWithData:imageData];
+            if (image) {
+                cachedImage = image;
+                GADMAdapterInMobiCacheSetObjectForKey(imageCache, cacheKey, cachedImage);
+            }
+        }
+        dispatch_async(dispatch_get_main_queue(), ^{
+            callback(cachedImage);
+        });
     });
-  });
 }
 
 #pragma mark - Completion
 
 - (void)notifyCompletion {
-  id<GADMAdNetworkConnector> strongConnector = _connector;
-  id<GADMAdNetworkAdapter> strongAdapter = _adapter;
-  if (!strongConnector || !strongAdapter) {
-    return;
-  }
-
-  if (!_mappedIcon || !_mappedImages) {
-    NSError *error = GADMAdapterInMobiErrorWithCodeAndDescription(
-        GADMAdapterInMobiErrorMissingNativeAssets,
-        @"InMobi returned an ad without image or icon assets.");
-    [strongConnector adapter:strongAdapter didFailAd:error];
-    return;
-  }
-
-  [strongConnector adapter:strongAdapter didReceiveMediatedUnifiedNativeAd:self];
+    if (!_mappedIcon || !_mappedImages) {
+        NSError *error = GADMAdapterInMobiErrorWithCodeAndDescription(
+                                                                      GADMAdapterInMobiErrorMissingNativeAssets,
+                                                                      @"InMobi returned an ad without image or icon assets.");
+        _nativeRenderCompletionHandler(nil,error);
+        return;
+    }
+    
+    _nativeAdEventDelegate = _nativeRenderCompletionHandler(self,nil);
+    
 }
 
 #pragma mark - Helpers
 
 - (BOOL)isValidWithNativeAd:(nonnull IMNative *)native imageURL:(nonnull NSString *)imageURL {
-  if (!native.adTitle.length || !native.adDescription.length || !native.adCtaText.length ||
-      !native.adIcon || !imageURL.length) {
-    return NO;
-  }
-  return YES;
```

Comment:
Can you revert all indentation or format changes? We will run our internal formatting tool after this PR is merged.